build.fsx

```diff
@@ -1,24 +1,129 @@
-// --------------------------------------------------------------------------------------
-// FAKE build script 
-// --------------------------------------------------------------------------------------
-#if FAKE
-#r "paket: groupref Build //"
-#endif
-#load "./.fake/build.fsx/intellisense.fsx"
+// TODO: delete this reference, it makes the script take ages to start
```

Comment:
I've left this in for now because the change is already massive. We still depend on FAKE's release-note parsing.

Reply:
Sure. I figured that would be the hardest to replace. Perhaps there is a simple exe replacement. At work we use GitVersion, don't know if it does this. Anyway, fine to leave this in.

Reply:
Yeah, NerdBank.GitVersioning should be pretty easy to drop in later.